doc/source/f2py/common_session.dat

```diff
@@ -1,27 +1,30 @@
 >>> import common
->>> print common.data.__doc__
-i - 'i'-scalar
-x - 'i'-array(4)
-a - 'f'-array(2,3)
+>>> print(common.data.__doc__)
+'i'-scalar
```

Comment:
Could you restore the name of common block items also here?

Reply:
Done, thanks!